### ISSUE TITLE
EDDI won't store the spoken name from one session to other.

### ISSUE THREAD

**Issue by fernandodandrea** [EDCD/EDDI#1725]
## EDDI version in which issue is found
3.5.2

## Steps to reproduce
In Commander Details, insert a phonetic name in the Name (spoken) box. Hear it to confirm it's ok.
Close EDDI.
Reopen it.

## Expected
Phonetic, spoken name, should last through sessions.

## Observed
The Name (spoken) field is blank.

**Comment by Tkael** [comment]
Hmm. The name is preserved for me. It's being written to `%appdata%/EDDI/eddi.json` and is being read when the app is re-opened.

Would you please confirm
1. That you have no other older versions of EDDI installed (since running an old version would likely drop that property from the config file) and
2. That `%appdata%/EDDI/eddi.json` has been updated with the correct PhoneticName after you close EDDI?

**Comment by fernandodandrea** [comment]
No other versions. File eddi.json was modified when I opened the tool, but not when it was closed. The spoken name wasn't saved.
It's contents are as follow:
`{
  "homeSystem": null,
  "homeStation": null,
  "destinationSystem": null,
  "destinationStation": null,
  "squadronName": null,
  "squadronID": null,
  "squadronRank": "None",
  "squadronAllegiance": "$faction_None",
  "squadronPower": "None",
  "squadronSystem": null,
  "squadronFaction": null,
  "debug": false,
  "beta": false,
  "plugins": {
    "Galnet monitor": false
  },
  "Gender": "Male",
  "powerMerits": null,
  "exporttarget": "Coriolis",
  "OverrideCulture": "en",
  "CommanderName": null,
  "PhoneticName": null
}`

**Comment by Tkael** [comment]
To be clear, you are saying that the file was modified correctly while EDDI was open but when EDDI was closed it changed back to null?

**Comment by fernandodandrea** [comment]
No, I inferred this from the modification date of the file. I just confirmed that, as soon as I open the tool, eddi.json, EDDI.sqlite and eddi.log era touched. The contents eddi.json remain the same.

eddi.json wasn't touched when I closed the tool yesterday. The modified date didn't change.

Today I made a test putting just "d" and closed the tool. It saved the "d" correctly to eddi.json. So I theorized the string I was using ("ˌdʲan̥ˈdʲʀɛːa") had invalid characters, so I put it in the tool to see if it changed to null in eddi.json and, surprised, the file was written perfectly.

I'm not sure why this has happened. Maybe the tool fails after working in tandem with the game for some hours? Maybe it crashes when I close it?

**Comment by Tkael** [comment]
When you now reopen EDDI, do you find the phonetic name correctly recorded?

**Comment by fernandodandrea** [comment]
Yes, it does.

But I also messed the tests: I set a home station. It might be the case
data wasn't saved correctly without a home station set?
